.github/workflows/release-gh-pages.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
```

Comment:
Shall we also use `persist-credentials: false` in  `actions/checkout` as suggested [here](https://github.com/cycjimmy/semantic-release-action#basic-usage)?

Reply:
yeap